docs/source/install/index.rst

```diff
@@ -2,16 +2,18 @@ Installation
 ============
 
 That's OK! You're busy, we get it. How do you just get started? Get yourself a clean 64-bit Linux
-box that fits the :doc:`system requirements <system_requirements>`, and run this command:
+box that fits the :doc:`system requirements <system_requirements>`. Make sure that ``curl`` is up to date
+using ``sudo apt-get install curl`` on Ubuntu, or ``sudo yum install curl nss`` on RHEL/CentOS. Then run
```

Comment:
It might be nice, but it's probably better not to distract people before the installation even happened. :)